packages/blade-coverage-extension/README.md

```diff
@@ -0,0 +1,33 @@
+# Blade Coverage Plugin
+
+Blade coverage plugin helps you in getting coverage of blade on a web page instantaneously.
+
```

Comment:
add video that you have attached in the announcement

Reply:
Added